update import documentation (#47)
snabble/docs: file_import_formats.md

## CSV Format

Using CSV, the following files have to be provided. It is possible to upload them in one tar.gz file:
* `products.csv`
* `codes.csv`

> * (`prices.csv`)

Reviewer: ```suggestion
* `prices.csv` (optional)
```

> The `prices.csv` is written in brackets because there are two ways of importing product prices: 

Reviewer: ```suggestion
The prices and discounted prices can be provided in two forms:
```

> 1. To support different prices for several stores we have a pricing category concept. Every product can be in more than one pricing category. At the beginning you may not be interested in using it and deliver `default` prices only but later you will be able to add pricing categories for other stores. In this case you have to deliver a `prices.csv`. The [Pricing API documentation](api_pricing.md) will tell you more about prices and pricing categories.

Reviewer: ```suggestion
1.  As columns in the `products.csv`
2. In an additional `prices.csv`. This allows to assign an individual prices and discounts to a product in a pricing category. Pricing categories are mapped to shops. This allows to assign individual prices and discounts to shops. See the [Pricing API documentation](api_pricing.md) for details.
```

> 2. The other way is to import one price per product within the `products.csv`. It is **not** possible to add more than one price per product to this file.

Reviewer: ```suggestion
```

File format:
* encoding: `UTF-8`
* value separator: `,`

### products.csv

> The `products.csv` contains one product per line. The [Products API documentation](api_products.md) will explain the structure of a product.

Reviewer: ```suggestion
The `products.csv` contains one product per line. The [Products API documentation](api_products.md) explains the structure of a product.
```

> The exact behavior of the importer and set of columns in the CSV interface have to be determined in the on-boarding process.

Reviewer: ```suggestion
The exact behavior of the importer and the set of columns in the CSV interface have to be determined in the on-boarding process.
```

Example CSV with prices:
```
#sku,name,taxCategory,depositProduct,bundledProduct,imageUrl,productType,price,discountedPrice
48,duplo white,normal,,,https://snabble.io/demodata/images/duplo-white-single.jpg,default,35,30
49,Deposit,normal,,,,deposit,8,
50,Deposit Six Pack,normal,,,,deposit,48,
51,Soda,normal,49,,,default,180,
52,Six Pack Soda,normal,50,51,,default,1080,1000
53,Apple,normal,,,,weighable,,
```

Example CSV without prices (`prices.csv` necessary):
```
#sku,name,taxCategory,depositProduct,bundledProduct,imageUrl,productType
48,duplo white,normal,,,https://snabble.io/demodata/images/duplo-white-single.jpg,default
49,Deposit,normal,,,,deposit
50,Deposit Six Pack,normal,,,,deposit
51,Soda,normal,49,,,default
52,Six Pack Soda,normal,50,51,,default
53,Apple,normal,,,,weighable
```

### codes.csv
The `codes.csv` contains the scannable EAN-8 or EAN-13 codes and refers to the product id.

> In the [Products API documentation](api_products.md#code-object) you will find more information on scannable codes.

Reviewer: ```suggestion
In the [Products API documentation](api_products.md#code-object) you find more information on scannable codes.
```
